src/strategies_and_params/reportingstrategy.jl

```diff
@@ -135,6 +135,14 @@ function report_after_step(::ReportingStrategy, args...)
     return nothing
 end
 
+"""
+    reporting_interval(::ReportingStrategy)
+
+Get the interval between steps for which non-essential statistics are reported. Defaults to
+1 if chosen `ReportingStrategy` does not specify an interval.
+"""
+reporting_interval(::ReportingStrategy) = 1
```

Comment:
```suggestion
"""
    reporting_interval(::ReportingStrategy)

Get the interval between steps for which non-essential statistics are reported. Defaults to
1 if chosen [`ReportingStrategy`](@ref) does not specify an interval.
"""
reporting_interval(::ReportingStrategy) = 1
```
Looks good.  :-)

In addition, please amend the docstring of `ReportingStrategy`, as we have now extended the interface:
```
# Interface:
A `ReportingStrategy` can define any of the following:
* [`refine_r_strat`](@ref)
* [`report!`](@ref)
* [`report_after_step`](@ref)
* [`finalize_report!`](@ref)
* [`reporting_interval`](@ref)
```

Also, please amend the docstring of `report_after_step`, as it currently states that "This function is called exactly once at the very end of a step.", which is no longer true.

Reply:
It would be good to also action the suggested change in the docstring of `ReportingStrategy`.